SingularityService/src/main/docker/start.sh

```diff
@@ -15,7 +15,12 @@ DEFAULT_URI_BASE="http://${SINGULARITY_HOSTNAME:=localhost}:${SINGULARITY_PORT:=
 
 args+=( -Xmx${SINGULARITY_MAX_HEAP:-512m} )
 args+=( -Djava.net.preferIPv4Stack=true )
+args+=( -Ddw.server.applicationContextPath="${SINGULARITY_APP_CTX:=/singularity}" )
```

Comment:
Given the changes in #1236 , think the config and vars can stay as-is. Tested this out and going to merge though our branches 👍